bundles/AdminBundle/Security/Guard/AdminAuthenticator.php

```diff
@@ -252,6 +253,9 @@ public function getUser($credentials, UserProviderInterface $userProvider)
             }
         }
 
+        // trigger LOGIN_SUCCESS event if user has been fully authenicated
+        $this->dispatcher->dispatch(AdminEvents::LOGIN_SUCCESS, new LoginSuccessEvent($user));
```

Comment:
this is not the right place for success event as it will be triggered on every getUser() call. please see https://symfony.com/doc/current/security/guard_authentication.html#step-2-create-the-authenticator-class

Please move it to the right place considering scenarios like 2FA, deeplink etc. Thanks!